pegasus/sites.v3/hourofcode.com/views/home_signup.haml

```diff
@@ -2,6 +2,7 @@
 -# We are hiding announcements in 2020
 - showAnnouncements = false
 %a#hocsignupform{name: 'signup'}
+#join-us-header=hoc_s(:front_join_us_button)
```

Comment:
Feel free to merge ahead of answering this but can you explain why this might fix the issue with the form?

Reply:
Yep! It's being referenced here: https://github.com/code-dot-org/code-dot-org/blob/39aaa1ae2ea8eba1b27af894dc9f74097c197dc2/apps/src/sites/hourofcode.com/pages/public/index.js#L190